src/cheshire/core.clj

```diff
@@ -8,9 +8,34 @@
                                        JsonGenerator
                                        JsonGenerator$Feature)
            (com.fasterxml.jackson.dataformat.smile SmileFactory)
+           (cheshire.prettyprint CustomPrettyPrinter)
            (java.io StringWriter StringReader BufferedReader BufferedWriter
                     ByteArrayOutputStream OutputStream Reader Writer)))
 
+(def ^:private default-pretty-print-options
+  {:indentation 2
+   :indent-arrays? false
+   :indent-objects? true
+   :before-array-values nil
+   :after-array-values nil
+   :object-field-value-separator nil})
+
+(defn- create-pretty-printer
```

Comment:
Sorry, I don't understand. You mean we should allow passing your own default options / pretty printer generation function? Isn't this already done with accepting `PrettyPrinter` instances in `:pretty`?